### ISSUE TITLE
Design Sysops icon

### ISSUE THREAD

**Issue by tcannonfodder** [tcannonfodder/houston#21]
Self-explanatory. Not sure what it's supposed to look like, but it should look like _something_ 😜

**Comment by tcannonfodder** [comment]
Also need icons for:
- Navigation
- Docking
- Vessel Overview
- Mission Wall
- Staging Analysis

**Comment by madrobby** [comment]
Do you accept PRs?

**Comment by tcannonfodder** [comment]
Absolutely! 😄 I started work on a Docking icon, but it still needs tweaking. I'm thinking of reusing FIDO icon (a compass) for Navigation. :)

I'm using Sketch for the icons, but can provide SVG versions if you want to use illustrator :)

![docking-620px](https://cloud.githubusercontent.com/assets/877224/16457651/36c7ca72-3dea-11e6-8b29-3b0a4797a84c.png)
![navigation-620px](https://cloud.githubusercontent.com/assets/877224/16457650/36c61970-3dea-11e6-9ffd-9e8e649b747e.png)
